lib/android/src/main/java/com/wix/reactnativeuilib/keyboardinput/CustomKeyboardLayout.java

```diff
@@ -167,16 +181,20 @@ private int getHeightForCustomContent() {
     }
 
     private void setKeyboardOverlayMode() {
-        Window window = getWindow();
-        if (window != null) {
-            window.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN);
+        if (mIsShown) {
+            Window window = getWindow();
+            if (window != null) {
+                window.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN);
+            }
         }
```

Comment:
Maybe extract this functionality to a separate method since it pretty much returns itself

Reply:
Yes, they are very similar, don't you think?

Reply:
ping

Reply:
I do think so, I usually try to minimize changes in these files; done.
Just to be sure it's not missed, there **_is_** a small risk in this PR.

Reply:
What's the risk?

Reply:
We have a few `Activities` that have a different `windowSoftInputMode` than `adjustResize` they might look different if this bug has indeed overridden their selection.
I can send you the details in DM.